include/deal.II/grid/grid_generator.h

```diff
@@ -1755,6 +1755,16 @@ namespace GridGenerator
     const bool                             copy_manifold_ids   = false,
     const std::vector<types::manifold_id> &manifold_priorities = {});
 
+  void
+  extrude_triangulation(
```

Comment:
Can you add a doxygen comment here please with something like:

Overload of extrude_triangulation() to allow dimension independent code to compile. This function throws an error when called as extrude_triangulation() is only implemented to extrude a dim=2 to a dim=3 Triangulation.

Reply:
I added some documentation. I thought `specialization` might be more appropriate than overload.

Reply:
These are technically overloads since they do not have any template parameters :)

Reply:
Technically I agree, but I would say the way it is used is rather in a sense of template specialization. Should I change it to overload then?

Reply:
To be clear on the resolution of this thread: the current text is overload so we're good :+1: